src/sentry/grouping/ingest/config.py

```diff
@@ -17,6 +17,17 @@
 
 Job = MutableMapping[str, Any]
 
+# We are moving all projects off these configuration without waiting for events
+CONFIGS_TO_DEPRECATE = []
+
+
+def upgrade_deprecated_configs():
+    queryset = Project.objects.filter(
+        projectoption__key="sentry:grouping_config", projectoption__value__in=CONFIGS_TO_DEPRECATE
+    )
+    for project in queryset:
+        _auto_update_grouping(project)
```

Comment:
This function is being modified to notice that the project is on a deprecated config, thus, upgrading it.